packages/suite/src/components/suite/TransactionsGraph/index.tsx

```diff
@@ -142,6 +142,7 @@ const TransactionsGraph = React.memo((props: Props) => {
                             />
                             <Tooltip
                                 position={{ y: 0, x: 0 }}
+                                wrapperStyle={{ 'z-index': '3' }}
```

Comment:
Probably too low

https://user-images.githubusercontent.com/33235762/133738599-9ab2c874-85d4-499e-910f-03305ae8eafd.mov

Reply:
> Probably too low
> Screen.Recording.2021-09-17.at.8.55.07.mov

@tomasklim imo its ok that sticky header covers another element on page as its navigation has higher priority? maybe lets discuss it with UI team to be sure?

Reply:
https://github.com/trezor/trezor-suite/issues/4227#issuecomment-933321948